### PR TITLE
feat: put something on the Getting Started page

### PR DESCRIPTION
**Pull request by aniravi24**

### CODE REVIEW

File: pages/docs/getting-started.mdx [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  Effect's docs are still very much a work in progress!
+</Callout>
+
+## Start Reading
+
+If you'd like to read more about Effect, start reading through the [Core Concepts](../core-concepts/effect.mdx)
+
+## Crashcourse
+
+If you're looking to dive right in, check out the [unofficial crashcourse](https://github.com/pigoz/effect-crashcourse/)
```
Comment (aniravi24) [inline pages/docs/getting-started.mdx:15]: Not sure if we wanted to link to this (or if we might even consider making this an "official" repo, just put this here for now as its the only way I would currently recommend anyone getting started with Effect. Let me know if this is fine

Comment (datner) [reply pages/docs/getting-started.mdx:15]: !!not an authority!!
I think this is fine, it's not as if theres an alternative and this crashcourse has proved invaluable

Comment (mikearnaldi) [reply pages/docs/getting-started.mdx:15]: I am not against linking it but what would be great is to absorb it, I know @pigoz planned to migrate it to the docs already, maybe a good opportunity

Comment (IMax153) [reply pages/docs/getting-started.mdx:15]: Yea honestly I think the less places an individual learning Effect has to "travel" to in order to get information, the better. 

I agree with @mikearnaldi that this is probably a good time to get content from the crash course into the docs site.

**Comment by IMax153** [comment]
Another thought, under the `Getting Started` section perhaps we can have a sub-page which discusses the `@effect` vs `effect` modules on NPM and how to install/use them.

File: pages/docs/getting-started.mdx (hunk 1 above)
Comment (aniravi24) [reply pages/docs/getting-started.mdx:15]: I agree that preferably we have it directly on the site. Where do you see the content going? Should it be directly incorporated into core concepts, or would it be a separate part of the docs?

**Comment by aniravi24** [comment]
> Another thought, under the `Getting Started` section perhaps we can have a sub-page which discusses the `@effect` vs `effect` modules on NPM and how to install/use them.

I'll definitely incorporate this somehow. People are confused by this often (understandably so) while new Effect is in this transition period.

File: pages/docs/getting-started.mdx (hunk 1 above)
Comment (aniravi24) [reply pages/docs/getting-started.mdx:15]: Could also put it under a section called crashcourse or overview or something, and then re-organize later.

**Comment by aniravi24** [comment]
> Another thought, under the `Getting Started` section perhaps we can have a sub-page which discusses the `@effect` vs `effect` modules on NPM and how to install/use them.

I added a small stub at the top of this of this file for now, can polish or move elsewhere later.

File: pages/docs/getting-started.mdx (hunk 1 above)
Comment (IMax153) [reply pages/docs/getting-started.mdx:26]: @aniravi24 - I think it should be directly included under core concepts. A lot of the content I already wrote there was inspired / ported from the crash course 😅 .

**Comment by IMax153** [comment]
> I added some copy mainly as a placeholder for this at the top of this file, for now, can polish or move elsewhere later.

I think this is fine for now, we can always add detail over time. The point is to just get some reasonable content out there (so thank you for helping with that :) )

File: pages/docs/getting-started.mdx (hunk 1 above)
Comment (aniravi24) [reply pages/docs/getting-started.mdx:26]: Ok sure, let me remove the crashcourse link for now and we can get this merged? Can do a separate PR on GitHub for starting working on a port if we decide to proceed there @IMax153

**Review by IMax153 — APPROVED** [review]

**Pull request merged**